### ISSUE TITLE
Add details on "lab booking calendar"

### ISSUE THREAD

**Issue by yarikoptic** [Donders-Institute/dicom-dataflow#9]
is it the endpoints configured in https://github.com/Donders-Institute/dicom-dataflow/blob/master/docker/cal2wl/config/config.yml 

NB Extract from an email correspondence

**Comment by marcelzwiers** [comment]
This is something that @hurngchunlee may answer best, but in general, I can say that we have developed an in-house portal in which scanner slots can be booked (calendar booking system). The project number, subject number and session number are taken from the portal project database and injected in the the DICOM worklist. You would probably need to implement a translation yourself from your in-house booking system to a worklist (which is then easily taken up by the worklist broker and pushed to any DICOM modality)

**Comment by hurngchunlee** [comment]
@yarikoptic the file you found out is a configuration template used by a program called `cal2wl`.  The source code of the program is in another repository.

In essence, the program is a tool that, on one hand, queries our lab-booking system (part of the project database, or pdb in short); while, on the other hand, transforms the query results into DICOM worklist. As @marcelzwiers mentioned, you will need to build this tool to adapt the data structure/interface of your local lab-booking system.

[This Go source code of `cal2wl`](https://github.com/dccn-tg/tg-toolset-golang/blob/master/dataflow/internal/cmd/dicom_worklist/generate.go) might be interesting/helpful as it contains the logic how we create the DICOM worklist.

Let me know if you need more information.

**Comment by yarikoptic** [comment]
Thank you @marcelzwiers and @hurngchunlee 

Do you have somewhere the writeup and/or schematic for all things you had automated and/or overall workflow , as how data curation happens, how processing etc.  May be resultant datasets (in BIDS?)

**Comment by hurngchunlee** [comment]
@yarikoptic I don't think we have a complete one.  This repo's README describes a workflow integrating DCCN's lab booking system, the MR console and image data transfer to the storage system from which further data processing (e.g. QC) can carry on by researchers.

@marcelzwiers maybe you know better than me about any automation after image data is delivered to the storage?
